streaming/java/streaming-runtime/src/test/java/io/ray/streaming/runtime/streamingqueue/StreamingQueueTest.java

```diff
@@ -160,7 +157,7 @@ public void testWordCount() {
     Map<String, Integer> wordCount = new ConcurrentHashMap<>();
     StreamingContext streamingContext = StreamingContext.buildContext();
     Map<String, String> config = new HashMap<>();
-    config.put(Config.CHANNEL_TYPE, Config.NATIVE_CHANNEL);
+    config.put(Config.CHANNEL_TYPE, "NATIVE_CHANNEL");
```

Comment:
lower case?

Reply:
Need upper case to transfer to Enum.
Config issue to next PR.